doc/manual/src/command-ref/nix-build.md

```diff
@@ -51,8 +51,9 @@ derivation).
 
 # Options
 
-All options not listed here are passed to `nix-store --realise`,
-except for `--arg` and `--attr` / `-A` which are passed to `nix-instantiate`.
+All options not listed here are passed to [`nix-store
+--realise`](@docroot@/command-ref/nix-store/realise.md),
+except for `--arg` and `--attr` / `-A` which are passed to [`nix-instantiate`](nix-instantiate.md).
```

Comment:
Nit, but I I think it is better to wrap this way
```suggestion
All options not listed here are passed to
[`nix-store --realise`](@docroot@/command-ref/nix-store/realise.md),
except for `--arg` and `--attr` / `-A` which are passed to [`nix-instantiate`](nix-instantiate.md).
```

we are trying to get towards punctuation-based wrapping, and when a sentence is too long because of a link, it is probably best to just keep the link (and any punctuation that cannot be separated, like that comma) on its own line.

Reply:
I also mildly prefer the relative path when no `..` is needed.

Reply:
Applied :)